templates/tfengine/components/resources/groups/main.tf

```diff
@@ -0,0 +1,30 @@
+{{- /* Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */ -}}
+
+{{range .groups}}
+module "{{resourceName . "id"}}" {
+  source  = "terraform-google-modules/group/google"
+  version = "~> 0.1"
+
+  id = "{{.id}}"
+  customer_id = "{{.customer_id}}"
```

Comment:
Couldn't these two also just be
```
{{hclField . "id" -}}
{{hclField . "customer_id" -}}
```

Reply:
`hclField` is for optional fields. `id` and `customer_id` are mandatory fields so use {{.id}} and {{.customer_id}} to make sure it is enforced.